app/src/main/java/com/teamounce/ounce/main/BottomSheetFragment.kt

```diff
@@ -27,44 +34,44 @@ class BottomSheetFragment : BottomSheetDialogFragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        prefs = SharedPreferences(view.context)
         //dialog?.window?.setBackgroundDrawableResource(R.drawable.bottom_sheet_box)
-        bottomSheetView()
+        bottomSheetView(view)
 
     }
 
     override fun getTheme(): Int = R.style.BottomSheetDialogTheme
 
 
-    fun bottomSheetView(){
-        bottomSheetAdapter = BottomSheetAdapter()
+    fun bottomSheetView(view: View){
+        bottomSheetAdapter = BottomSheetAdapter(view.context)
```

Comment:
```suggestion
        bottomSheetAdapter = BottomSheetAdapter(requireContext())
```

Fragment에서 Context 가져올 땐 requireContext 함수로 해결 가능